src/packages/fetch-engine/src/download.ts

```diff
@@ -129,6 +129,10 @@ export async function download(options: DownloadOptions): Promise<BinaryPaths> {
       }),
   )
 
+  if (process.env.BINARY_DOWNLOAD_VERSION) {
+    options.version = process.env.BINARY_DOWNLOAD_VERSION
+  }
+
```

Comment:
New feature inside another feature? 😄 Does someone in docs knows or maybe we need to create an issue about that to document it?

Sidenote: Is it tested?